hack/update-helm-index.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash -e
+
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o pipefail
+
+asset_urls="$@"
+
+git checkout gh-pages
```

Comment:
Check https://github.com/NVIDIA/k8s-device-plugin/pull/688/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R303-R308 where I checkout a local copy of the repo in some other folder to avoid conflicts with the branch that is currently checked out.

Reply:
done